unityfiles/Assets/Scripts/Game/Menu/PauseMenu.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PauseMenu : Singleton<PauseMenu> {
```

Comment:
Hvorfor singleton på denne?

Reply:
Godt spørsmål. Skal se om jeg husker det. Tror bare det kan tas bort

Reply:
Om den strengtatt trenger å være en Singleton vet jeg ikke, men gjorde det for at det var litt lettere å lazy loade klassen når jeg skal endre state